Security/Risk_management/Rapid_Risk_Assessment.mediawiki

```diff
@@ -0,0 +1,356 @@
+** Ask again and specify you want to make sure they've listed all sensitive data they can think of.
+* Set the data classification for each data type in the dictionary, such as "PUBLIC", "CONFIDENTIAL", etc. Mozilla uses standard classification levels.
+* If some compensating controls are mentioned or there are more details about the data you can fill them in as well.
+** For example "User attributes, classified INTERNAL, protected by LDAP authentication, used to find out the user
+t-shirt size".
+* Based on this dictionary/catalog of data, figure out which is the main data classification the service is handling. If unsure, ask the team.
+** For example "mock is mainly used to serve package files that are PUBLIC." (public main data classification).
+** Set this in the header section at the top of the RRA template.
+
+=== RRA Risk table (5-10 minutes) ===
+
+This is where the risk impact assessment is done, and where the probability for these impact is roughly estimated.
+The risk impact is using ''probable impact'', that is, worst-case scenario impacts that seem possible.
+
+==== Levels summary ====
+
+===== Reputation =====
+
+{| class="wikitable"
+|-
+| <span style="padding: 0 32px;background-color:lightgrey;">LOW</span>
+| Nobody cares.
+|-
+| <span style="padding: 0 20px; background-color:steelblue;">MEDIUM</span>
+| Tweets, forum posts, e-mails are seen.
+|-
+| <span style="padding: 0 32px;background-color:#FFF380;">HIGH</span>
+| Articles in technical websites such as HN, Ars-Technica, etc. are seen.
+|-
+| <span style="padding: 0 14px; background-color:#E77471;">MAXIMUM</span>
+| Scandal. We appear on TV, BBC, etc.
+|-
+|}
+
+===== Productivity =====
+
+{| class="wikitable"
+|-
+| <span style="padding: 0 32px;background-color:lightgrey;">LOW</span>
+| SG (Small Group): affected for < 24h. LG (Large Group): affected for minutes.
+|-
+| <span style="padding: 0 20px; background-color:steelblue;">MEDIUM</span>
+| SG: affected for days. LG: affected for hours.
+|-
+| <span style="padding: 0 32px;background-color:#FFF380;">HIGH</span>
+| SG: affected for weeks. LG: affected for days.
+|-
+| <span style="padding: 0 14px; background-color:#E77471;">MAXIMUM</span>
+| SG: affected for a month or more. LG: affected for weeks.
+|-
+|}
+
+===== Finances =====
+
+{| class="wikitable"
+|-
+| <span style="padding: 0 32px;background-color:lightgrey;">LOW</span>
+| < 100k USD.
+|-
+| <span style="padding: 0 20px; background-color:steelblue;">MEDIUM</span>
+| < 1M USD.
+|-
+| <span style="padding: 0 32px;background-color:#FFF380;">HIGH</span>
+| < 10M USD.
+|-
+| <span style="padding: 0 14px; background-color:#E77471;">MAXIMUM</span>
+| > 10M USD.
+|-
+|}
+
+Work-force (productivity-derived) financial costs calculations are included in the RRA spreadsheet.
+
+==== Recording risk impacts ====
+
+
+The risk impact is solely sourced from the RRA when risk is caculated. Thus, the RRA risk impact is the most important
```

Comment:
"caculated"
maybe "cactus-lated"
although I'm not sure it's cactus related at all...